client/registries.go

```diff
@@ -23,34 +18,41 @@ func (c *Client) AddRegistry(server, username, password, email string) (*Registr
 		"serveraddress": server,
 	}
 
-	var registry Registry
-
-	err := c.Post("/registries", params, &registry)
-
+	system, err := c.GetSystem()
 	if err != nil {
 		return nil, err
 	}
 
-	return &registry, nil
-}
+	// backwards compatible
+	// FIXME: pick the right version here
+	if system.Version < "30000000000000" {
```

Comment:
Yes, I'll fix it in a quick CLI release once I have a release number for this release.